SasquatchTV/SasquatchTV/MobileCenterDelegate.swift

```diff
@@ -0,0 +1,60 @@
+/**
+ * Protocol for interacting with MobileCenter SDK.
+ * SasquatchSwift and SasquatchObjC implement this protocol
```

Comment:
I know this file is from Sasquatch app. Can you revise this comment? It should be enough to say `Swift and Objective C implement ...` in Sasquatch. Then we can easily copy-and-paste it from Sasquatch to mac and tv apps.
Also change it in SasquatchMac if you copy-and-pasted.